Topics/User_Experience.md

```diff
@@ -2,7 +2,7 @@
 
 ### Overview
 
-User experience is a large and broad topic. It covers the overall experience of the user and how they interact with software systems and applications. The confusing part is that UI and UX design, while related can also be totally different. For example, UI could be how the interface is designed whereas UX is more about how users interact with the said design.
+User experience is a large and broad topic. It covers the overall experience of the user and how they interact with software systems and applications. The main focus of User Experience is to make assessible, easy to follow, user-friendly, and be adaptable to the user's needs. Although, there is no set standard rules that make up a "Great" User Experience, it often differs on the product and the consumers that are using the products. It is very often confused with UI (User Interface) and UX design, while related can also be totally different. For example, UI could be how the interface is designed whereas UX is more about how users interact with the said design.
```

Comment:
Grammatical errors here. In the sentence, "The main focus of User Experience is to make assessible, easy to follow, user-friendly, ...", do you mean accessible instead of assessible? Similarly, you don't indicate **what** is being made "assessable, easy to follow, user-friendly". 

Change the grammar issues in "Although, there is no set standard rules that make up a "Great" User Experience, it often differs on the product and the consumers that are using the products". One possibility is, "There is no set of standard rules that make up "great" user experience, as it often differs on the product and the consumers that are using the products." 

When you mention "User Experience" and "User Interface" for the first time, you can instead write "User Interface (UI)" and "User Experience (UX)", which will allow you to write UI/UX in the following sentences without changing them. Then, in the sentence "It is very often confused with UI (User Interface) and UX design, while related can also be totally different", UI can be used without adding "(User Interface). Also, this sentence does not really make sense. Consider changing it to, "User Experience is not to be confused with UI and UX Design. While these topics have similarities with User Experience, they are also different in their own ways."

Reply:
Looks good